### PR TITLE
docs(PPDSC-2703): update form stories

### PR DESCRIPTION
**Pull request by mstuartf**
[PPDSC-2703](https://nidigitalsolutions.jira.com/browse/PPDSC-2703)

**What**

1. Background - why this is needed
2. What did you do
3. What does the reviewers should expect

**I have done:**
- [ ] Written unit tests against changes
- [ ] Written functional tests against the component and/or NewsKit site
- [ ] Updated relevant documentation

**I have tested manually:**
- [ ] The feature's functionality is working as expected on Chrome, Firefox, Safari and Edge
- [ ] The screen reader reads and flows through the elements as expected.
- [ ] There are no new errors in the browser console coming from this PR.
- [ ] When visual test is not added, it renders correctly on different browsers and mobile viewports (Safari, Firefox, small mobile viewport, tablet)
- [ ] The Playground feature is working as expected

**Before:**

**After:**

**Who should review this PR:**

**How to test:**

[PPDSC-2703]: https://nidigitalsolutions.jira.com/browse/PPDSC-2703?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by pp-serviceaccount** [comment]
You can preview these changes on:
  Documentation Site: http://ncu-newskit-docs-pr.s3-website-eu-west-1.amazonaws.com/ppdsc-2703-form-stories
  Storybook: http://ncu-newskit-docs-pr.s3-website-eu-west-1.amazonaws.com/ppdsc-2703-form-stories/storybook

File: src/form/__tests__/form.stories.tsx [hunk 1]
```diff
@@ -31,436 +39,513 @@ const validateUserName = async (value: string) => {
...
-        <Button type="submit">Submit</Button>
-      </Form>
-    </Block>
-  </>
+const Container = styled.div`
+  max-width: 300px;
+`;
+
+export const StoryFormDefault = () => (
+  <Container>
```
Comment (mutebg) [inline src/form/__tests__/form.stories.tsx:89]: Can you use StorybookPage as a wrapper to stories

File: src/form/__tests__/form.stories.tsx [hunk 2]
```diff
@@ -31,436 +39,513 @@ const validateUserName = async (value: string) => {
...
+    border: 1px solid #919191;
+    border-radius: 8px;
+    padding: 12px;
+    margin-bottom: 8px;
+  }
+
+  .PhoneInput:has(input:focus) {
+    outline: 2px solid #3768fb;
+  }
+`;
```
Comment (mutebg) [inline src/form/__tests__/form.stories.tsx:220]: I wonder if we should keep this example, it was created a long time ago just to demonstrate how can be used with this 3rd party library.

Comment (mstuartf) [reply src/form/__tests__/form.stories.tsx:220]: @GeriReid @nathanparris what do you think?

Comment (GeriReid) [reply src/form/__tests__/form.stories.tsx:221]: @mstuartf include it if it's useful to engineers.

File: src/form/__tests__/form.stories.tsx [hunk 3]
```diff
@@ -31,436 +40,503 @@ const validateUserName = async (value: string) => {
...
+          <Button type="submit">Submit</Button>
+        </GridLayoutItem>
+      </GridLayout>
+    </Form>
+  </StorybookPage>
+);
+StoryFormPhoneInputExample.storyName = 'Phone input example';
+
+export const StoryFormEmailInputExample = () => (
+  <Container>
```
Comment (mutebg) [inline src/form/__tests__/form.stories.tsx:278]: I think is good to use Storybookpage on every story in that way you will have consistent padding around components
Screenshot 2023-03-30 at 6 54 03

Comment (mstuartf) [reply src/form/__tests__/form.stories.tsx:278]: missed this one, fixed

**Review by mutebg — APPROVED** [review]

**Review by JohnTParsons — APPROVED** [review]

**Pull request merged**